src/Command/Dispatcher/DispatchSettingsCommand.php

```diff
@@ -88,12 +88,8 @@ private function updateRepositories(Project $project): void
         ];
 
         if ($project->hasBranches()) {
-            $branchNames = $project->branchNames();
-            $defaultBranch = end($branchNames);
-
-            if (\is_string($defaultBranch)) {
-                $infoToUpdate['default_branch'] = $defaultBranch;
-            }
+            $defaultBranch = $project->stableBranch() ?? $project->unstableBranch();
+            $infoToUpdate['default_branch'] = $defaultBranch->name();
         }
```

Comment:
Can we add a "defaultBranch()" method to Project?

Including a testcase?

Reply:
Done